.github/ISSUE_TEMPLATE/bug_template.yaml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug]
+body:
+ - type: markdown
+   attributes:
+    value: |
+     Thanks for taking the time to fill out this bug report!
+ - type: textarea
+   id: what-happened
+   attributes:
+    label: What happened?
+    description: Also tell us, what did you expect to happen?
```

Comment:
We can have something like `Acceptance Criteria` or `Expected behavior` as one more mandatory. Helps us to get a clean requirement and expectation of the customer

Reply:
+1